js/util.js

```diff
@@ -112,3 +112,25 @@ export const reduceCount = (count, digits = 1) => {
   // Do not reduce if we are below 1000
   return divisor ? (count / divisor).toFixed(digits) + indicator : count
 }
+
+export const relativeTime = time => {
```

Comment:
There's actually already a relative util in this file, called `prettyTime`. 
Is there an advantage to this one or can we just use the existing one?